docs/projects/blank/tutorial/0/key-pieces.md

```diff
@@ -0,0 +1,156 @@
+| ----- | ------------------------------------------------------------------------------ |
+| HTTPS | `git clone https://github.com/spatialos/gdk-for-unity-blank-project.git` |
+| SSH   | `git clone git@github.com:spatialos/gdk-for-unity-blank-project.git`     |
+
+> You can only clone via SSH if you have already [set up SSH keys (GitHub)](https://help.github.com/articles/connecting-to-github-with-ssh/) with your GitHub account.
+
+<%(/Expandable)%>
+
+## Configurations
+
+At the root of the project you’ll find two launch configurations; one for local deployments and one for cloud deployments.
+
+```text
+
+    gdk-for-unity-blank-project/
+        ├── snapshots/
+        ├── workers/
+        ├── cloud_launch.json
+        ├── default_launch.json
+        ├── spatialos.json
+
+```
+
+These configurations are used to determine what template your deployment uses, the world size, the load balancing policies, and some worker-specific configuration parameters.
+
+The snapshots directory contains a default snapshot defining the initial state of your game world.
+
+```text
+
+    gdk-for-unity-blank-project/
+        ├── snapshots/
+                ├── default.snapshot
+        ...
+
+```
+
+You can find the root of the Unity project in `workers/unity/`.
+
+```text
+
+    gdk-for-unity-blank-project/
+        ...
+        ├── workers/
+            ├── unity/
+                ├── Assets/
+                ├── Packages/
+                ├── ProjectSettings/
+                ├── spatialos.UnityClient.worker.json
+                ├── spatialos.UnityGameLogic.worker.json
+        ...
+
+```
+
+Here, you will find the client-worker and server-worker configurations. They are used to configure how a worker should be launched and specify particular Runtime settings for the given worker type.
+
+Under `Assets/Config/` of the Unity Project, there is a `BuildConfiguration` asset.
+
+```text
+
+    Assets/
+        ├── Config/
+            ├── BuildConfiguration.asset
+            ...
+
+```
+
+This asset defines for each worker type the target platforms to build and which Scenes to include. By default the GDK builds both Windows and MacOS UnityClients for cloud deployments, but only build a single platform (Windows or MacOS depending on your machine) for local deployments. This is because you will only ever run a client for local deployments on your machine, but may want to distribute your game to both Windows and MacOS users when running a cloud deployment.
+
+See below to learn more about:
+
+* [SpatialOS launch configurations](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/project-layout/launch-config)
+* [SpatialOS worker configurations](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/project-layout/introduction)
+* [Snapshots]({{urlRoot}}/reference/concepts/snapshots)
+* [GDK worker build configurations]({{urlRoot}}/modules/build-system/build-config)
+
+## Scenes
```

Comment:
At this point, should we direct users to open the Unity project? Since we are inspecting Unity things from here on out, it may be better to do it in the context of the engine

Reply:
good idea 👍